app-emulation/hercules-sdl-crypto/hercules-sdl-crypto-4.8.0.ebuild

```
@@ -0,0 +1,20 @@
+# Copyright 2022-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+# Use ../hercules-sdl/files/gen_hashes.sh to identify the relevant
+# commit when tagging new versions.
+COMMIT="9ac58405c2b91fb7cd230aed474dc7059f0fcad9"
+
+DESCRIPTION="Simple AES/DES encryption and SHA1/SHA2 hashing library"
+HOMEPAGE="https://github.com/SDL-Hercules-390/crypto"
+SRC_URI="https://github.com/SDL-Hercules-390/crypto/archive/${COMMIT}.tar.gz -> crypto-${COMMIT}.tar.gz"
```

I think this should probably be:
```suggestion
SRC_URI="https://github.com/SDL-Hercules-390/crypto/archive/${COMMIT}.tar.gz -> ${PN}-${COMMIT}.tar.gz"
```
as, though there's a hash there, `crypto-` isn't super descriptive as a distfile name.